.ci/cirrus.runner.yml

```
@@ -7,11 +7,11 @@ task:
         XCODE_VERSIONS: "16.1"
         DISK_SIZE: 190
       - MACOS_VERSION: sequoia
-        XCODE_VERSIONS: "16.4,\"26-rc\",16.3,16.2,16.1,16"
+        XCODE_VERSIONS: "26,16.4,26,16.3,16.2,16.1,16"
```

Version 26 is duplicated
```suggestion
        XCODE_VERSIONS: "26,16.4,16.3,16.2,16.1,16"
```